docs/general/serializers.md

```diff
@@ -217,7 +217,15 @@ The object being serialized.
 
 #### #root
 
-PR please :)
+Allows you to change the root object name of the payload.
+
+e.g.
+
+```ruby
+def root
+  "posts"
+end
```

Comment:
@jimbeaudoin well, the thing is that `root` at this time is an options that is passed into the serializer.  The method you would want to override is actually `json_key` or even set the `type`.  (That's why I referenced the other pr which describes some of this in the diff).  I'd certainly merge this change as an option to setting the resource root, but am hesitant for the documentation to advocate an unintended usage...